src/index.html

```diff
@@ -7,8 +7,93 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="icon">
+          <a href="#Home">
+            <img
+             src="images\logo.png"
+             alt="logo"
+             class="logo">
+          </a>
+        </div>
+        <nav class="nav">
+
+          <ul class="nav__list">
+
+            <li class="nav__item">
+              <a href="#Apple"
+               class="is-active nav__link"
+              >
+                Apple
+              </a>
+            </li>
```

Comment:
```suggestion
        </div>
        
        <nav class="nav">
          <ul class="nav__list">
            <li class="nav__item">
              <a href="#Apple" class="is-active nav__link">
                Apple
              </a>
            </li>
```
Don't leave new lines between parent and child element. Please fix all cases